src/Build.OM.UnitTests/Definition/Project_Tests.cs

```diff
@@ -3739,6 +3739,21 @@ public void GetAllGlobsShouldFindAllExcludesAndRemoves()
         new[] {"aa", "bb", "cc"},
         new[] {"b", "c"}
         )]
+        [InlineData(
+            @"<A Include=`ab*;bc|*;de*`/>",
+            new[] {"ab", "de"},
+            new[] {"bc", "bc|"}
+            )]
+        [InlineData(
+            @"<A Include=`\ab*;bc*;de*`/>",
```

Comment:
This one worries me: `\ab*` is a valid glob (all files in the root of the current drive that start with `ab`).